lab_3_keywords_textrank/main.py

```diff
@@ -51,7 +52,10 @@ def _clean_and_tokenize(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens
         """
-        pass
+        for i in text:
```

Comment:
why iterating over all symbols in text? consider iterating over punctuation symbols only